### DISCUSSION TITLE
A way to drop series from Keep Reading/On Deck

### DISCUSSION THREAD

**Discussion by stonechitlin** [gotson/komga#1550] [Ideas]
### Describe your suggested feature

A way to drop series from Keep Reading/On Deck, perhaps with an extra value per item to designate if it should show up on the home page or not in those lists. As it stands the only way to "drop" a series it to essentially reset your current progress on it by marking the entire thing as unread or read.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by gotson** [comment]
Duplicate of #599

**Comment by stonechitlin** [comment]
Apologies, I was only searching under Open tasks to see if it was already listed.

**Comment by nicolasshu** [comment]
Any chance this can be reopened?

**Comment by HydraulicGoose** [comment]
To add to this, a simple "mark as dropped" button would be good.
<img width="346" height="285" alt="image" src="https://github.com/user-attachments/assets/dd28daeb-29fe-461e-8692-35e3f5bbc060" />
